src/Playwright/WebSocketFrame.cs

```diff
@@ -1,9 +1,10 @@
 using System;
 using System.Text;
 
+#pragma warning disable CS1591, SA1600
```

Comment:
something like that might work for the entire solution/project instead of adding it to every file: https://docs.microsoft.com/de-de/previous-versions/visualstudio/visual-studio-2015/ide/how-to-suppress-compiler-warnings?view=vs-2015&redirectedfrom=MSDN#suppressing-warnings-for-visual-basic

Reply:
Yeah, but I think I only want it for internal namespaces :/